greymatter/inputs.cue

```diff
@@ -1,62 +1,94 @@
 package greymatter
 
-config: {
-	// enable Spire-based mTLS
-	spire: bool | *false @tag(spire,type=bool)
-}
-
+// WARNING - Configurations in the mesh section must not be changed
+// arbitrarily because a tenant's mesh configurations hook into an existing
+// enterprise's core mesh configurations. Change these values in accordance
+// with enterprise-level configurations. This may require coordination with
+// an enterprise operations team.
```

Comment:
```
// WARNING - Values in the mesh section must not be changed
// arbitrarily because a tenant's project hooks into an existing
// enterprise's mesh. Change these values in accordance
// with enterprise-level direction. This may require coordination with
// an operations team.
```
Maybe this reads a bit cleaner?

Reply:
I like it.

Reply:
Why don't you use suggestions?

Reply:
I couldn't use the suggestions here cause it was a big block. Apparently you can only do suggestions on one line? I tried 🤷‍♂️

Reply:
You just click and drag down for multiline.